packages/esm-commons-lib/src/api.resource.ts

```diff
@@ -191,7 +183,9 @@ export function fetchPatientLastEncounter(patientUuid: string, encounterType) {
 }
 
 export function fetchPatientCovidOutcome() {
-  return openmrsFetch(`/ws/rest/v1/reportingrest/cohort/${covidOutcomesCohortUUID}`).then(({ data }) => {
+  return openmrsFetch(
```

Comment:
Is it possible to also turn this into a hook

Reply:
Actually, this endpoint is not in use anymore